datasets/UKNI01_northern_ireland/UKNI01_northern_ireland.derived.ad

```diff
@@ -18,26 +18,25 @@
 - has_mountains = true
 - has_old_technologies = true
 - has_other = true
-- has_solar_csp = false
-- has_offshore_solar = false
+- has_solar_csp = true
 - has_other_emissions_section = true
 - has_import_export = true
-- use_network_calculations = false
+- use_network_calculations = true
 - use_merit_order_demands = true
 - has_weather_curves = false
-- has_aggregated_chemical_industry = true
-- has_detailed_chemical_industry = false
+- has_aggregated_chemical_industry = false
+- has_detailed_chemical_industry = true
 - has_coal_oil_for_heating_built_environment = true
-- buildings_insulation_employment_constant = 0.0
+- buildings_insulation_employment_constant = 0.0093
 - households_insulation_employment_constant = 0.014
 - co2_emission_1990_aviation_bunkers = 15.37818186
 - co2_emission_1990_marine_bunkers = 8.69679035
-- co2_percentage_free = 0.85
-- co2_price = 0.005
-- offshore_ccs_potential_mt_per_year = 42.5
-- economic_multiplier = 0.0
-- employment_fraction_production = 0.0
-- employment_local_fraction = 0.0
+- co2_percentage_free = 0.4
+- co2_price = 0.02483
```

Comment:
This CO2 price is linked to the UK 2019 dataset, while the NI dataset is for 2018. I suppose this applies to other values as well, is this an issue or is this as good as it gets, because we don't have a UK 2018 dataset?

Reply:
This is the CO2 price and percentage of the `eu` dataset and this is the same as the nl2019 dataset. So indeed the year doesn't correspond exactly, but we indeed do not have a 2018 UK dataset